crates/full-node/sov-sequencer/src/preferred/replica/event_handler.rs

```diff
@@ -48,13 +48,29 @@ where
 {
     #[allow(clippy::match_same_arms)]
     async fn on_db_event(&self, data: DbData) -> Result<(), DBDataRejected> {
-        match data {
-            DbData::BatchStart(_) => {}
-            DbData::Transaction(_, _, _) => {}
-            DbData::BatchEnd(_) => {}
-            DbData::NewProof => {}
+        let res: Result<(), ReplicaError<S>> = match data {
+            DbData::BatchStart(_) => Ok(()),
```

Comment:
In the next PR we will plug `process_do_batch_start_replica/process_do_new_tx_replica/process_close_current_batch_replica` here